App can deploy into the new CAS namespace

tahb opened the pull request:
The new community-accommodation-dev namespace is being defined here [1]. This change updates the existing Approved Premises configuration that we copied across and gets temporary-accommodation ready to launch into the new namespace.

Whilst the API service is referenced, it is not deployed into this new namespace yet so some cooridination will be needed.

We found that this line `tlsSecretName: temporary-accommodation-ui-cert # override per environment` includes a value that we can't trace. We think it's a default that if it were used would cause issues, however in values-dev.yml this key is being set to `hmpps-temporary-accommodation-dev-cert` which is known. We aren't sure if we can remove the former without causing an issue so we leave it the same as AP for now.

[1] https://github.com/ministryofjustice/cloud-platform-environments/pull/9081

bryangaledxw commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Fork the frontend](https://trello.com/c/Dwvx44O0/477-fork-the-frontend)

tahb committed:
App can deploy into the new CAS namespace

The new community-accommodation-dev namespace is being defined here [1]. This change updates the existing Approved Premises configuration that we copied across and gets temporary-accommodation ready to launch into the new namespace.

Whilst the API service is referenced, it is not deployed into this new namespace yet so some cooridination will be needed.

We found that this line `tlsSecretName: temporary-accommodation-ui-cert # override per environment` includes a value that we can't trace. We think it's a default that if it were used would cause issues, however in values-dev.yml this key is being set to `hmpps-temporary-accommodation-dev-cert` which is known. We aren't sure if we can remove the former without causing an issue so we leave it the same as AP for now.

[1] https://github.com/ministryofjustice/cloud-platform-environments/pull/9081

tahb commented on helm_deploy/values-dev.yaml:
```diff
@@ -6,14 +6,14 @@ generic-service:
     ENVIRONMENT: dev
-    APPROVED_PREMISES_API_URL: 'https://approved-premises-api-dev.hmpps.service.justice.gov.uk'
-    INGRESS_URL: 'https://approved-premises-dev.hmpps.service.justice.gov.uk'
+    APPROVED_PREMISES_API_URL: 'https://cas-approved-premises-api-dev.hmpps.service.justice.gov.uk'
```
This app needs to be launched.

tahb commented on helm_deploy/hmpps-temporary-accommodation-ui/values.yaml:
```diff
@@ -42,7 +42,7 @@ generic-service:
 
   namespace_secrets:
-    approved-premises-ui:
+    temporary-accommodation-ui:
```
These secrets need to be written when you have a moment to show me how please @pezholio

tahb committed:
Ask CircleCI to deploy to the dev environment

We'll need to configure CircleCI next so that it knows to deploy to the recently added hmpps-community-accommodation-dev namespace.
